src/styles/content.scss

```diff
@@ -186,4 +186,36 @@
     background: #cfcfcf;
     border: 0;
   }
+
+  .tooltip {
```

Comment:
was initially there.. moved it to here to denote that its available for all components.